packages/dds/tree/.vscode/Tree.code-workspace

```diff
@@ -21,12 +21,14 @@
 		"typescript.preferences.autoImportSpecifierExcludeRegexes": [
 			// Avoid suggesting imports from parent index file, which almost always is a mistake which creates a cyclic dependency.
 			// Importers in almost all cases should import it from a sibling instead (the same place the index file imports it from).
-			"^\\.\\/index\\.js",
+			"^\\.\\/index\\.js$",
```

Comment:
this change didn't seem necessary, but being more specific like this is probably good practice.